shared/pingpong/pingpong.go

```diff
@@ -186,6 +186,10 @@ func RunPingPong(ctx context.Context, ac libgoal.Client, accounts map[string]uin
 func sendFromTo(fromList, toList []string, accounts map[string]uint64, assetParams map[uint64]v1.AssetParams, client libgoal.Client, cfg PpConfig) (sentCount, successCount uint64, err error) {
 	amt := cfg.MaxAmt
 	fee := cfg.MaxFee
+	if cfg.NumAsset > 0 && len(assetParams) <= 0 {
```

Comment:
This check should happen before here, i.e. after or even within preprareAccount. In addition, failing this check should abort the entire pingpong program rather than make a single send fail.